### DISCUSSION TITLE
Edit watched date in Plex

### DISCUSSION THREAD

**Discussion by glensc** [Taxel/PlexTraktSync#1626] [Ideas]
With the new social in plex where you can share activity with your friends, I noticed you can change the watched date of events.

<img width="488" alt="image" src="https://github.com/Taxel/PlexTraktSync/assets/199095/f2638a98-7327-4d1a-9995-2d38a02fd931">

I wonder can how update watched date timestamp on plex side too, currently the timestamp for "add play" was always current time

**Comment by simonc56** [comment]
This Activity watched date doesn't seem to be the same than the one in PMS session/history.
Changing this activity date doesn't change the PMS session/history read by PlexTraktSync.

Further tests are needed though.
If user remove Movie in PMS, then re-add it, maybe the activity date will [sync](https://support.plex.tv/articles/sync-watch-state-and-ratings/) to PMS watched date...

**Reply by glensc** [reply]
It would be a new feature, sync activity watched date.
